### DISCUSSION TITLE
[Projects Beta] Allow descriptions for a project

### DISCUSSION THREAD

**Discussion by Relequestual** [community/community#6366] [Projects and Issues] [answered]
Projects looks great.
I would like to be able to set a description for a project in a simialr way you can for a milestone.

Giving context is important as it allows you to communicate why the project exists.
This might include linking to a discussion or issue that spawend the project!

**Comment by azenMatt** [comment, marked as answer]
Hi @Relequestual - we just added this 🎉 https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/
